doc-content/enterprise-only/openshift/startupstrategy-enable-proc.adoc

```diff
@@ -10,15 +10,23 @@ You
 endif::DM[]
 can scale the {KIE_SERVER} pod, but all the copies execute the same services.
 
-If you want to connect additional {KIE_SERVERS} to {CENTRAL}, you must enable the `OpenShiftStartupStrategy` setting in the environment. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.
+You can connect additional {KIE_SERVERS} to {CENTRAL}. However, if you deployed a single authoring environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml`, you must enable the `OpenShiftStartupStrategy` setting in the environment. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.
 
-If you enable the `OpenShiftStartupStrategy` setting, an additional limitation applies to your environment. If a user is deploying a service to the {KIE_SERVER}, no user is able to deploy another service to the same {KIE_SERVER} until the first deployment completes. Because of this limitation, the `OpenShiftStartupStrategy` setting might not be suitable for some authoring environments. 
+With the `OpenShiftStartupStrategy` setting, when a user deploys a service to the {KIE_SERVER}, the {KIE_SERVER} deployment is rolled out again. No user is able to deploy another service to the same {KIE_SERVER} until the roll-out completes. Because the roll-out might take noticeable time, the `OpenShiftStartupStrategy` setting might not be suitable for some authoring environments. 
+
+Do not complete this procedure if you deployed a high-availability authoring environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml` template. In this environment, the `OpenShiftStartupStrategy` setting is enabled by default.
 
 Do not complete this procedure unless you want to connect additional {KIE_SERVERS} to {CENTRAL}.
 
+[NOTE]
+====
+In {PRODUCT} version 7.7.0 only, the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring-ha.yaml` template did not enable the `OpenShiftStartupStrategy` setting. If you deployed this version, you must complete this procedure to enable the `OpenShiftStartupStrategy` setting. The high-availability authoring environment requires this setting for stable operation of a single {KIE_SERVER} or of multiple {KIE_SERVERS}.
+====
```

Comment:
I would add "the"

The high-availability authoring environment requires this setting for the stable operation of a single {KIE_SERVER} or of multiple {KIE_SERVERS}.